eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/http/RequestUriAndTimeLogging.java

```diff
@@ -56,8 +59,16 @@ public RequestUriLogging(HttpRequestExecutor delegate, boolean enableLog, String
     {
         if (mEnableLog)
         {
-            Log.d(mTag, String.format("%s %s", request.method().verb(), uri));
+            int requestId = sRequestCounter.getAndIncrement();
+            Log.d(mTag, String.format("(%s->) %s %s", requestId, request.method().verb(), uri));
+            long start = SystemClock.uptimeMillis();
+            T result = mDelegate.execute(uri, request);
+            Log.d(mTag, String.format("(%s<-) Response time: %s ms", requestId, SystemClock.uptimeMillis() - start));
+            return result;
+        }
+        else
+        {
+            return mDelegate.execute(uri, request);
```

Comment:
that's no good style, calling the the delegate twice in different branches makes it hard to read and easy to get wrong. I'd prefer a branch before and after the sole call to the delegate.